source/config/configSpec.py

```diff
@@ -13,7 +13,7 @@
 #: provide an upgrade step (@see profileUpgradeSteps.py). An upgrade step does not need to be added when
 #: just adding a new element to (or removing from) the schema, only when old versions of the config
 #: (conforming to old schema versions) will not work correctly with the new schema.
-latestSchemaVersion = 18
+latestSchemaVersion = 19
```

Comment:
This is an interesting case. Spec 17 is equal to 19. I wonder whether we should deal with this separately in some way, i.e. when the spec version is 17, do nothing when upgrading to 19, thus ignore the upgrade steps but just increase the number.

May be we should support something like
`def upgradeConfigFrom_17_to_19(profile: ConfigObj) -> None: ...`

Then change the logic of the upgrader in such a way that it will search an upgrade function based on version 17, and when that yields two functions, take the function that has the highest `to`, 19 in this case.